src/handlers/context-menu-handler.ts

```diff
@@ -0,0 +1,16 @@
+import type { ContextMenuCommandInteraction } from 'discord.js';
+
+export default async function messageContextMenuHandler(interaction: ContextMenuCommandInteraction): Promise<void> {
+	const { commandName } = interaction;
+
+	const contextMenus = interaction.client.contextMenus;
+	const contextMenu = contextMenus.get(commandName);
+
+	// do nothing if no context menu
+	if (!contextMenu) {
+		throw new Error(`Missing command handler for \`${commandName}\``);
+	}
+
+	// run the context menu
```

Comment:
```suggestion
	// * Do nothing if no context menu
	if (!contextMenu) {
		throw new Error(`Missing command handler for \`${commandName}\``);
	}

	// * Run the context menu
```

Reply:
I've updated this and also fixed it in the command handler